spec/conversions.md

```diff
@@ -303,7 +303,7 @@ End Module
 
 The output of the program is:
 
-```
+```console
```

Comment:
As far as i know there is no Markdown language `console`. Did you mean `bash`? See https://highlightjs.org/static/demo/

Reply:
That's right, it is not a language. However, "console" is on the list of [supported language IDs](https://review.docs.microsoft.com/en-us/help/contribute/code-in-docs?branch=master#supported-languages). In order to put a descriptive ID on these code blocks and avoid a P0 CATS report error, we have been using "console" for examples of output that are displayed to the console (like in this case), and for examples of command line tools.

Reply:
Alright, agree, better than nothing.